cc.retroshare.retroshare-gui.yaml

```diff
@@ -103,13 +118,16 @@ modules:
           - CONFIG+=release
           - CONFIG+=no_sqlcipher
   sources:
-  - type: archive
-    url: https://github.com/RetroShare/RetroShare/archive/v0.6.5.zip
-    sha256: e6d78eb5fb1dcc179f561cbe4934ef6f242a97be966e00c0a6d0bcde7e034183
+  - type: git
+    url: https://github.com/RetroShare/RetroShare.git
+    tag: v0.6.6
+    commit: 751fffc30f957425b1581c3aefaacf1959a8869f
   - type: file
     path: cc.retroshare.retroshare-gui.appdata.xml
```

Comment:
You need to update the version in the appstream file since it's is part of this repository.